### ISSUE TITLE
Upgrading an old 2.x sqlite DB to 3.0 can cause tables to be truncated

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#5349]
Upgrading an old 2.x sqlite database to 3.0 can cause the tables to be truncated: 

![screenshot 2016-05-22 11 37 57](https://cloud.githubusercontent.com/assets/1833361/15453322/d55c3c6e-2011-11e6-81c2-cec5790e84e8.png)

Details
-------

 - Relevant Bolt Version: [  3.0 | master ]

**Comment by bobdenotter** [comment]
Some further investigation: 

The issue seems to be in the `templatefields` column. Adding these before hitting the "update db" button makes it work properly: 

```
ALTER TABLE "bolt_pages" ADD "templatefields" text NOT NULL DEFAULT '';
ALTER TABLE "bolt_showcases" ADD "templatefields" text NOT NULL DEFAULT '';
```

**Comment by bobdenotter** [comment]
Perhaps this is fices by #5362 as well?

**Comment by bobdenotter** [comment]
Yup, fixed by #5632.

**Comment by GawainLynch** [comment]
:+1:
